### ISSUE TITLE
Pulling in Files Correctly

### ISSUE THREAD

**Issue by gotchosevan** [avocode/avocode#288]
Not bringing in files properly. PSD layers don't get imported.
-----

```
TypeError: Cannot read property 'x' of null
  at c.calculateBounds (/Applications/Avocode.app/Contents/Resources/app/node_modules/avocode-select-tool/lib/select-tool.js:15:284)
  at c.mousemoveDrag (/Applications/Avocode.app/Contents/Resources/app/node_modules/avocode-select-tool/lib/select-tool.js:9:367)
  at /Applications/Avocode.app/Contents/Resources/app/node_modules/avocode-select-tool/lib/select-tool.js:1:140
  at c.module.exports.Emitter.emit (/Applications/Avocode.app/Contents/Resources/app/node_modules/emissary/lib/emitter.js:118:15)
  at c.mousewheel (/Applications/Avocode.app/Contents/Resources/app/node_modules/avocode/lib/stage.js:23:423)
  at HTMLCanvasElement.<anonymous> (/Applications/Avocode.app/Contents/Resources/app/node_modules/avocode/lib/stage.js:1:324)

```

**Comment by rychlis** [comment]
Could you please give us more information about your operating system? Does it happen in all PSDs? It is probably fixed in 2.2.0, let me know if you still experience this issue

**Comment by rychlis** [comment]
As this should be fixed in 2.2.1 I'm closing this issue. If you experience it again in the latest version, let me know and we will reopen the issue.
